config.yml

```diff
@@ -43,7 +43,19 @@ common_settings:
       key: A_VERY_COMMONLY_USED_PASSWORD
       value: "ThisIsAVerySecretPassword"
       variable_type: env_var #or file
-      protected: false #
+      protected: "false" #
```

Comment:
I think that this is a GitLab bug. :( But I would rather work around it in the code instead of asking users to do this counterintuitive "bool as string" thing. Can you code it please?

Reply:
Also: I was not able to reproduce this issue, please see https://github.com/egnyte/gitlabform/issues/107#issuecomment-653876243 .

Reply:
I tested in the latest release and confirmed it works.